tests/Content/ContentTranslationTest.php

```diff
@@ -109,9 +109,18 @@ public function testContentMerged()
 	public function testContentFile()
 	{
 		$app = new App([
+			'languages' => [
```

Comment:
What was the reasoning to remove those props (making the app object single-lang in the test)?

Reply:
Is this the correct line? I'm converting the App object to a multi-language installation here.

Reply:
Dafuq. Either I was super confused or it showed up red before in the iOS app.